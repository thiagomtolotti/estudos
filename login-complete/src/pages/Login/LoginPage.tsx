"use client";

import Link from "next/link";
import styles from "./login.module.css";

import LoginForm from "./LoginForm";

export function LoginPage() {
	return (
		<div className={styles.login}>
			<h1>Login</h1>

			<LoginForm />

			<Link href={"/login/forgotPassword"} data-testid="forgot-password">
				<button className="button-clear">Esqueci minha senha</button>
			</Link>
		</div>
	);
}
