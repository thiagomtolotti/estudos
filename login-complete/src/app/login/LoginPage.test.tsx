import { RenderResult, render } from "@testing-library/react";
import Login from "./page";

describe("Login Page", () => {
	let container: RenderResult;
	beforeEach(() => {
		container = render(<Login />);
	});

	it("Should render correctly", () => {
		expect(container.container).toMatchSnapshot();
	});
});
