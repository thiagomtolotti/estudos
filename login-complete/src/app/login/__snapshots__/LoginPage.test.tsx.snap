// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page Should render correctly 1`] = `
<div>
  <div
    class="login"
  >
    <h1>
      Login
    </h1>
    <form
      class="loginForm"
    >
      <input
        placeholder="email"
        type="email"
      />
      <input
        placeholder="senha"
        type="text"
      />
      <span
        class="rememberMe"
      >
        <input
          id="remember-me"
          name="remember-me"
          type="checkbox"
        />
        <label
          for="remember-me"
        >
          Lembre de mim
        </label>
      </span>
      <button>
        Login
      </button>
    </form>
    <button
      class="button-clear"
    >
      Esqueci minha senha
    </button>
  </div>
</div>
`;
